docs/docs/jupyter-integration.ipynb

```diff
@@ -0,0 +1,334 @@
+    "```python\n",
+    "app.upload(\"My App\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d38e6ac-b84d-44c4-a232-7861c3dbd8f0",
+   "metadata": {},
+   "source": [
+    "### Opt-in or opt-out"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "363c9d6e-7b43-4d68-9450-ffbeac167955",
+   "metadata": {},
+   "source": [
+    "By default, notebook-to-app conversion will include all markdown and supported cell output. This is the easiest and quickest way to turn your notebook into an app. \n",
+    "\n",
+    "In this mode, you can explicitly exclude cells with the cell tag `dp-exclude`. \n",
+    "\n",
+    "![Exclude cells](img/jupyter/exclude.png)\n",
+    "\n",
+    "In some cases, it may be preferable to only include cells of interest. This can be achieved by setting `opt_out` to `False`, e.g.:\n",
+    "\n",
+    "```python\n",
+    "app = dp.App(blocks=dp.cells_to_blocks(Out, opt_out=False))\n",
+    "```\n",
+    "\n",
+    "In this mode, you can explicitly include supported cells with the cell tag `dp-include`.\n",
+    "\n",
+    "![Include cells](img/jupyter/include.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be31381b-cd4a-4820-81c4-4d2ba357b59b",
+   "metadata": {},
+   "source": [
+    "### Display input code\n",
+    "\n",
+    "You may want to display the input code for a particular cell in your app. Datapane has you covered. You can include code cells with the cell tag `dp-show-code`.\n",
+    "\n",
+    "![Show code](img/jupyter/show-code.png)\n",
+    "\n",
+    "Let's do this for the cell below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e0ac95b7-1523-45c0-aca7-0fdfba249c5f",
+   "metadata": {
+    "tags": [
+     "dp-show-code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "dp.Embed(url='https://www.youtube.com/watch?v=_KS_yZBI71s&t')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d2e9759-b5db-4fe6-b58c-e1b10a39b21a",
+   "metadata": {},
+   "source": [
+    "## Supercharge your notebook with Datapane\n",
+    "\n",
+    "Datapane Blocks are powerful - and they can be embedded straight into your notebook.\n",
+    "\n",
+    "### Interactive DataFrames\n",
+    "\n",
+    "The notebook journey often begins by loading in a dataset and displaying it for a quick sanity check. Something like the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "edf62892-dda1-4b3e-b958-b77c74be6d10",
+   "metadata": {
+    "tags": [
+     "remove_input"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "dp.Media(file=\"img/jupyter/dataframe-output.png\", name=\"dataframe-output\", caption=\"Default DataFrame output behaviour\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "846d9177-84d0-4121-a165-cf5f9f41e150",
+   "metadata": {},
+   "source": [
+    "That's not bad, but it could be much better. Let's wrap our DataFrame in Datapane's `dp.DataTable` block and see what happens."
```

Comment:
To confirm, this will embed the interactive DataTable below?

Reply:
That's right - here is the docs page with it in view

<img width="792" alt="Screenshot 2022-10-28 at 12 22 22" src="https://user-images.githubusercontent.com/15690380/198575747-3bee60b7-1db7-4028-94a3-c498d78ef16f.png">